.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  latest_python: "3.11"
+  supported_pythons: "3.8|3.9|3.10|3.11"
+  miniforge_version: "22.9.0-2"
+  miniforge_variant: "Mambaforge"
```

Comment:
Did not know this was a thing. Very excited that this is a thing.

Reply:
Right? It sets up everything in a sane way as well.